README.md

```diff
@@ -10,24 +10,32 @@
 # cromshell
  A script for submitting workflows to a cromwell server and monitoring / querying their results.
 
-requires `column`, `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
-
-Current version: 2.0.0 
+Current version: 2.0.0.beta 
 
 ## Examples:
 
 ```
-         cromshell-alpha submit workflow.wdl inputs.json options.json dependencies.zip
-         cromshell-alpha status
-         cromshell-alpha -t 20 metadata
-         cromshell-alpha logs -2
+         cromshell-beta submit workflow.wdl inputs.json options.json dependencies.zip
+         cromshell-beta status
+         cromshell-beta -t 20 metadata
+         cromshell-beta logs -2
 ```
 
 ## Supported Flags:
+  * `--hide_logo`
```

Comment:
```suggestion
  * `--no_turtle`. or `--🐢🚫` --I_hate_turtles
```

Reply:
We should support all of those options.  Unfortunately it doesn't seem like any fonts support the combining cancellation character with the turtle emoji which is really the ideal name.